Cylenium/Element.cs

```diff
@@ -95,6 +95,27 @@ public Element Check()
             return this;
         }
 
+        /// <summary>
+        /// Uncheck the checkbox or radio element.
+        /// </summary>
+        /// <returns>The current element.</returns>
+        public Element Uncheck()
+        {
+            var type = WebElement.GetAttribute("type");
+            if (type != "checkbox" && type != "radio")
+                throw new UnexpectedTagNameException($"Element.Check() expects a 'checkbox' or 'radio' element, but got {type}");
+
+            if (this.IsChecked())
+            {
+                WebElement.Click();
+            }
+            else
+            {
+                // do nothing - already unchecked
+            }
```

Comment:
This is a bad example from me haha. Having empty blocks should be avoided.

```
if (IsChecked())
{
    WebElement.Click();
}

return this;
```

Reply:
haha lazy programming from me too. good point.